### ISSUE TITLE
How to set recurrence_duration?

### ISSUE THREAD

**Issue by jdietrch** [ad-si/TaskLite#5]
https://tasklite.org/concepts.html describes the `Recurring` secondary state, associated with the `recurrence_duration` field:
```
Task which needs to be done every day, week, etc. I.e. missed
completions must be caught up immediately. (e.g. paying rent)
```

How can I set up a monthly paying rent task?

The `repeat` command isn't suitable for at least two reasons:
1.) it sets `repetition_duration`, and not `recurrence_duration`
2.) it takes a certain number of days, and a monthly interval cannot be represented that way

**Comment by ad-si** [comment]
Unfortunately `tasklite recur` is not yet implemented. I'll bump its priority and let you know when its implemented!

**Comment by ad-si** [comment]
I finally implemented in 8028190 🎉
Feedback would be highly appreciated!
You can download the latest binaries at https://github.com/ad-si/TaskLite/actions/runs/76594083
